__tests__/index.test.js

```diff
@@ -25,7 +42,7 @@ describe("Purchases", () => {
     const eventInfo = {
       productIdentifier: "test.product.bla",
       purchaserInfo: purchaserInfoStub,
-      error: null,
+      error: null
```

Comment:
I find that prettier with the option "trailing commas: all" is better, since it makes it faster to add new elements to objects, arrays..